crosstool/cc_toolchain_config.bzl

```diff
@@ -2664,7 +2664,7 @@ please file an issue at https://github.com/bazelbuild/apple_support/issues/new
 
     features = [
         # Marker features
-        feature(name = "archive_param_file", enabled = True),
+        feature(name = "archive_param_file"),
```

Comment:
The rules_cc one is using `enabled = True` but this removes it, just checking that was expected

Reply:
Yea looks like this did the opposite of the description?

Reply:
Ohh, dang. Let me revert this. Not sure where the mixup came.

Reply:
Was a little confused because of https://github.com/bazelbuild/apple_support/pull/338